database/postgresql/seed.js

```diff
@@ -0,0 +1,29 @@
+const { Pool } = require('pg');
+const faker = require('faker');
+
+const pool = new Pool({
```

Comment:
you might wanna move this to another file. when you deploy you dont want your datbase info to be public